src/engine/N3Base/N3UIImage.cpp

```diff
@@ -35,6 +35,7 @@ CN3UIImage::~CN3UIImage() {
         delete[] m_pAnimImagesRef;
         m_pAnimImagesRef = NULL;
     }
+    m_szTexFN = "";
```

Comment:
I think you can remove this from the destructor. In the Release function it makes sense in case one wants to release existing resources, but not destroy the object. However, in a destructor since `std::string` does not need manual deallocation, then it's not needed.
```suggestion
```

Reply:
Thanks for the info! Agree. Please see commit: https://github.com/ko4life-net/ko/pull/125/commits/e9f6f91a35b77135f861e57c5d5affc5929c8ba4

Reply:
Glad to see someone actually using git [--fixup](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---fixupamendrewordltcommitgt) commits. This makes the review process so pleasant. LGTM.